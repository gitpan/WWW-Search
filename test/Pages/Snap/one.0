<!-- c|net Fri Oct  8 08:06:33 1999  $Revision: 2.29 $  -->

 








<html>
<head>
<title>Snap:Power Search:Results</title>
</head>

<body bgcolor="#FFFFFF" link="#0000FF" vlink="#666666" text="#333333">



<!-- c|net Mon Oct  4 15:34:52 1999  $Revision: 2.29 $  -->


<map name=gs-tabs><area shape=rect coords="2,22,81,44" href="/main/door/0,1,home-home,00.html?st.sn.0.t.fd"><area shape=rect coords="82,27,155,44" href="/main/local/1,37,home-8,00.html?tag=st.sn.0.t.lc"><area shape=rect coords="156,27,230,44" href="/LMOID/mysnap/mysnap/0,160,home-0,00.html?st.sn.0.t.ms"></map>
<table width="100%" cellspacing=0 cellpadding=0 border=0><tr bgcolor="#cc0000">
<td width="33%"><a href="/main/door/0,1,home-home,00.html"><img src="/main/images/logos/logo_header_fl.gif" alt=Snap.com width=185 height=45 border=0></a></td>
<td width="34%" align=center valign=bottom><img src="/main/images/tabs/none_fl_tgl.gif" alt="Navigation Tabs" width=234 height=45 border=0 usemap="#gs-tabs"></td>
<td width="33%"><spacer type=block width=1 height=1></td>
</tr><tr><td bgcolor="#666666" height=2 colspan=3><spacer type=block width=1 height=1></td></tr></table>
<table bgcolor="#FFFFFF" width="100%" cellpadding=0 cellspacing=0 border=0>
<tr valign=middle><td height=100 align=center><a href="http://home.snap.com/cgi-acc/acc_clickthru.acc?clickid=0000477f5811944e00000000&adt=banner&edt=home&cat=s-web&site=SN" target="_top"><img src="http://home.snap.com/Ads/Media/Images/snap.rsnwhistlerban.banner.gif?u_vid_0_0&ffffffff" alt="Brought to you by Snap.com and Resort Sports Network" border="2" height="60" width="468" ><br><center><font face="Verdana, Arial, Helvetica" size="-2">Brought to you by Snap.com and Resort Sports Network</font></center></A></td></tr></table>
<table width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td bgcolor="#0000FF" height=1><spacer type=block width=1 height=1></td></tr></table>
<table cellspacing=0 cellpadding=4 border=0 width="100%"><tr valign=top><td nowrap width="75%"><font size=-1 face="Arial,Helvetica">
Find books
about '<a
href="http://home.snap.com/cgi-acc/acc_clickthru.acc?clickid=0000479c3eb331ba00000000&adt=commercelink1&edt=home&cat=s-web&site=SN&url=http%3A%2F%2Fbn%2Ebfast%2Ecom%2Fbooklink%2Fclick%3Fsourceid%3D1380915%26categoryid%3Dmerchsearch%26match%3Dexact%26options%3Dand%26keyword%3Dvbthread">vbthread</a>'
at barnesandnoble.com.
</font></td><td width="25%" align=right nowrap><font size=-2 face="Arial,Helvetica">A D V E R T I S E M E N T</td></tr></table>

<table width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td bgcolor="#666666" height=2><spacer type=block width=1 height=1></td></tr>
<tr><td>
<table width="100%" cellpadding=4 cellspacing=0 border=0>
<tr bgcolor="#CCCCCC" valign=bottom><td nowrap>
<form method=get action="/search/directory/results/1,61,home-0,00.html">
<input type=text name=keyword value="+vbthread" size=40><input type=hidden name="SM" value="1">
<input type=hidden name="tag" value="st.sn.srsb.top">
&nbsp;&nbsp;<input type=submit value="Search">
&nbsp;&nbsp;<span style="color:#0000FF"><a href="/main/help/item/0,11,home-6736,00.html"><font color="#0000FF" face="Arial,Helvetica" size=-2>Search Tips</a></font></span></font></td></tr>
</form></table>
</td></tr></table>



    

<table align=right cellpadding=4 cellspacing=0 border=0>
<tr><td><font face="Arial,Helvetica" size=-2>A D V E R T I S E M E N T</font></td></tr>

<tr><td align=center>
<spacer type=block width=1 height=1></td></tr>

<tr><td align=center>
<spacer type=block width=1 height=1></td></tr>

<tr><td align=center>
<a href="http://home.snap.com/cgi-acc/acc_clickthru.acc?clickid=000047bbee99194b00000000&adt=midas1&edt=home&cat=s-web&site=SN" target="_top"><img src="http://home.snap.com/Ads/Media/Images/fs.88x31gottagetitfree.button..gif" alt="Free Stuff" border="1" height="31" width="88" ><br><center><font face="Verdana, Arial, Helvetica" size="-2">Free Stuff</font></center></A><hr noshade size=1></td></tr>

<tr><td align=center>
<spacer type=block width=1 height=1></td></tr>

<tr><td><spacer type=block height=1 width=1></td></tr>
</table>

<table width="75%" cellpadding=0 cellspacing=0 border=0>
<tr><td height=5 colspan=2><spacer type=block width=1 height=1></td></tr>


<tr><td height=5 colspan=2><spacer type=block width=1 height=1></td></tr>
<tr><td bgcolor="#CCCCCC" colspan=2><b><font face="Arial,Helvetica" size=-1>&nbsp;All Web Pages</font></b></td></tr>
</table>

<table width="100%" cellpadding=2 cellspacing=0 border=0>
<tr><td height=8><spacer type=block width=1 height=8></td></tr>
<tr valign=middle>
<td align=right width=101><a href="http://www.inktomi.com/"><img src="/main/images/sponsors/inktomi.gif" width=101 height=25 border=0></a></td>
<td nowrap width="100%"><font face="Arial,Helvetica" size=-2>Powered by <a href="http://www.inktomi.com/">Inktomi</a></font></td>
</tr></table>


<ul><font face="Arial,Helvetica" size=-1>
   
   
    
         
     <b><a href="http://redirect.inktomi.com/click?u=http://ftp.sunet.se/pub3/vendor/microsoft/Softlib/MSLFILES&q=02%3F4i%7Dkwmz~%7B%3FL%5E%5BUJLK7y%25vqtl~yz%40hpm%7Bl%3F3%3Fi%252.1-////6%3FL%5E%5BUJLK7y%25svqtysjg%3F3%3Fi%25/1/--//6&r=0&d=www&i=56&n=G2EU4TPNVNSS4RS5&s=0&c=">SUNET's Index of /pub3/vendor/microsoft/Softlib/MSLFILES</a></b><br>
     The World's #1Archive, Click Below: /pub What's New Search freetext Search filenames Feedback About AccessGuide More About SUNET Index of /pub3/vendor/microsoft/Softlib/MSLFILES Name Last modified Size Description Parent Directory 07-Sep-1999  16..<br>
   <font size=-2>Found at: http://ftp.sunet.se/pub3/vendor/microsoft/Softlib/MSLFILES</font><p>
    
         
     <b><a href="http://redirect.inktomi.com/click?u=http://www2.rad.kumc.edu/share&q=02%3F4i%7Dkwmz~%7B%3FL%5E%5BUJLK7y%25vqtl~yz%40hpm%7Bl%3F3%3Fi%252.1-////6%3FL%5E%5BUJLK7y%25svqtysjg%3F3%3Fi%25/1/--//6&r=1&d=www&i=56&n=G2EU4TPNVNSS4RS5&s=0&c=">No Title</a></b><br>
     August 9, 1999 S40135 Mfcdao36libs.exe PRB: Unrecognized Database Format Error w/ Access 2000 Database August 4, 1999 S40134 Dirlv.exe SAMPLE: DirLV Sample Populates a ListView Control Similar to Explorer August 3, 1999 S40133 Recover.exe  Excel..<br>
   <font size=-2>Found at: http://www2.rad.kumc.edu/share</font><p>
    
         
     <b><a href="http://redirect.inktomi.com/click?u=http://rs2.ch.liv.ac.uk/~dlc/Microsoft.txt&q=02%3F4i%7Dkwmz~%7B%3FL%5E%5BUJLK7y%25vqtl~yz%40hpm%7Bl%3F3%3Fi%252.1-////6%3FL%5E%5BUJLK7y%25svqtysjg%3F3%3Fi%25/1/--//6&r=2&d=www&i=56&n=G2EU4TPNVNSS4RS5&s=0&c=">No Title</a></b><br>
     July 27, 1999 S40132 Opropsheet.exe SAMPLE: Opropsheet.exe Create a Modeless OLE Property Sheet Using MFC July 27, 1999 S40131 Jetvc.exe PATCH: JetVC.exe VC++ Support Files for the Jet OLE DB Provider 4.0 July 23, 1999 S40130 Marshal.exe How  To..<br>
   <font size=-2>Found at: http://rs2.ch.liv.ac.uk/~dlc/Microsoft.txt</font><p>
    
         
     <b><a href="http://redirect.inktomi.com/click?u=http://ap01.physik.uni-greifswald.de/~ftp/internet/cica95.idx&q=02%3F4i%7Dkwmz~%7B%3FL%5E%5BUJLK7y%25vqtl~yz%40hpm%7Bl%3F3%3Fi%252.1-////6%3FL%5E%5BUJLK7y%25svqtysjg%3F3%3Fi%25/1/--//6&r=3&d=www&i=56&n=G2EU4TPNVNSS4RS5&s=0&c=">No Title</a></b><br>
     ** Der naechste 'Spiegel' des Cica-Verzeichnisses befindet sich ** in Rostock: ftp.uni-rostock.de ** Statt '~ftp/pub/pc/win95/*' heissen die Verzeichnisse 'ftp/Win95/cica/*'. * (Diese Datei wird woechentlich automatisch aktualisiert.) ** ftpphys  .<br>
   <font size=-2>Found at: http://ap01.physik.uni-greifswald.de/~ftp/internet/cica95.idx</font><p>
       </font></ul>
<table width="100%" cellpadding=3 cellspacing=0 border=0>
<tr>
<td align=right width=101><a href="http://www.inktomi.com/"><img src="/main/images/sponsors/inktomi.gif" width=101 height=25 border=0></a></td>
<td width="100%" nowrap><font face="Arial,Helvetica" size=-2>Powered by <a href="http://www.inktomi.com/">Inktomi</a></font></td></tr></table>
<b><font face="Arial,Helvetica" size=-1>
 
Page&nbsp;&nbsp;

  
      1&nbsp;&nbsp;
   of&nbsp;&nbsp;1 
</font></b><p>



     

<table width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td bgcolor="#666666" height=2><spacer type=block width=1 height=1></td></tr></table>


<!-- c|net Tue Oct  5 18:32:19 1999  $Revision: 2.29 $  -->


<table width="100%" cellpadding=4 cellspacing=0 border=0>
<tr><td><font face="Arial,Helvetica" size="-1" color="#0000FF">
<a href="/LMOID/resource/0,566,-409,00.html?ad.snbottom.0.cruise" style="color:#0000FF">
<font color="#0000FF">Win a Cruise -- Enter Today!</font></a></td>
<td align=right><font face="Arial,Helvetica" size="-2">A D V E R T I S E M E N T</font></td></tr></table>



<table width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td bgcolor="#0000FF" height=1><spacer type=block width=1 height=1></td></tr></table>

<table bgcolor="#FFFFFF" width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td height=100 align=center><a href="http://home.snap.com/cgi-acc/acc_clickthru.acc?clickid=0000477f5811944e00000000&adt=banner&edt=home&cat=s-web&site=SN" target="_top"><img src="http://home.snap.com/Ads/Media/Images/snap.rsnwhistlerban.banner.gif?u_vid_0_0&ffffffff" alt="Brought to you by Snap.com and Resort Sports Network" border="2" height="60" width="468" ><br><center><font face="Verdana, Arial, Helvetica" size="-2">Brought to you by Snap.com and Resort Sports Network</font></center></A></td></tr>
</table>

<!-- search box -->
<table bgcolor="#CCCCCC" width="100%" cellpadding=4 cellspacing=0 border=0>
<tr bgcolor="#CCCCCC"><td><form action="/search/power/results/1,180,home-0,00.html" method="get">
<font face="Arial,Helvetica" size="-2">Enter search word(s):</font><br><input type="text" name="KW" value="+vbthread" size="40">
<input type="hidden" name="KM" value="a">
<input type="hidden" name="AM0" value="m">
<input type="hidden" name="AT0" value="w">
<input type="hidden" name="AK0" value="">
<input type="hidden" name="AM1" value="">
<input type="hidden" name="AT1" value="">
<input type="hidden" name="AK1" value="">
<input type="hidden" name="AM2" value="">
<input type="hidden" name="AT2" value="">
<input type="hidden" name="AK2" value="">
<input type="hidden" name="AM3" value="">
<input type="hidden" name="AT3" value="">
<input type="hidden" name="AK3" value="">
<input type="hidden" name="AM4" value="">
<input type="hidden" name="AT4" value="">
<input type="hidden" name="AK4" value="">
<input type="hidden" name="AN" value="1">
<input type="hidden" name="NR" value="100">
<input type="hidden" name="FR" value="f">
<input type="hidden" name="PL" value="a">
<input type="hidden" name="DR" value="0">
<input type="hidden" name="FM" value="1">
<input type="hidden" name="FD" value="1">
<input type="hidden" name="FY" value="">
<input type="hidden" name="TM" value="">
<input type="hidden" name="TD" value="">
<input type="hidden" name="TY" value="">
<input type="hidden" name="LN" value="">
<input type="hidden" name="MT" value="">
<input type="hidden" name="XT" value="">
<input type="hidden" name="DM" value="">
<input type="hidden" name="SDI" value="">
&nbsp;&nbsp;<input type="submit" value="Search">&nbsp;&nbsp;<font face="Arial,Helvetica" size="-2"><a href="/main/help/item/0,11,home-6736,00.html">Search Tips</a></font>
<br>

<a href="/search/power/form/0,179,home-0,00.html"><font face="Arial,Helvetica" size=-1><b>New Search</b></font></a>&nbsp;|&nbsp;<a href="/search/power/form/1,179,home-0,00.html?KM=a&KW=%2Bvbthread&AM0=m&AT0=w&AK0=&AM1=&AT1=&AK1=&AM2=&AT2=&AK2=&AM3=&AT3=&AK3=&AM4=&AT4=&AK4=&AN=0&NR=100&FR=f&PL=a&DR=0&FM=1&FD=1&FY=&TM=&TD=&TY=&LN=&MT=&XT=&DM=&SDI="><font face="Arial,Helvetica" size=-1><b>Revise Search</b></font></a><br><br>


</td><td width=1><spacer type=block width=1 height=5></form></td></tr></table>
<table bgcolor="#CCCCCC" width="100%" cellpadding=0 cellspacing=0 border=0>
<tr><td><font face="Arial,Helvetica" size="-2">&nbsp;Or try one of these search engines:</font></td></tr>
<tr><td align=center><font face="Arial,Helvetica" size=-1>
<a href="/go/to?tag=ex.sn.sr.6.av&destUrl=http%3A%2F%2Fwww%2Esearch%2Ecom%2FAltaVista%2F1%2C57%2C0%2C0200%2Ehtml%3Fmode%3Dsimple%26what%3Dweb%26suffix%3D%26query%3D%252Bvbthread%26format%3D2">AltaVista</a> | 
<a href="/main/finance/news/search/1,216,home-0,00.html?tag=st.sn.sr.6.bbn&keyword=%2Bvbthread">Bloomberg News</a> |
<a href="/go/to?tag=ex.sn.sr.6.dn&destUrl=http%3A%2F%2Fx8%2Edejanews%2Ecom%2Fdnquery%2Exp%3FQRY%3D%252Bvbthread%26defaultOp%3DAND%26svcclass%3Ddncurrent%26maxhits%3D20%26ST%3DQS%26format%3Dterse%26DBS%3D2">DejaNews</a> | 
<a href="/go/to?tag=ex.sn.sr.6.xcit&destUrl=http%3A%2F%2Fwww%2Eexcite%2Ecom%2Fsearch%2Egw%3Fsearch%3D%252Bvbthread%26searchType%3DKeyword%26category%3Ddefault%26mode%3Drelevance%26showqbe%3D1%26display%3Dhtml3%2Chb">Excite</a> | 
<a href="/go/to?tag=ex.sn.sr.6.goto&destUrl=http%3A%2F%2Fwww%2Egoto%2Ecom%2Fd%2Fsearch%2Fp%2Fsnap%2F%3FKeywords%3D%252Bvbthread"><b>GoTo</b></a> | 
<a href="/go/to?tag=ex.sn.sr.6.ipix&destUrl=http%3A%2F%2Fisurf%2Einterpix%2Ecom%2Fcgi%2Dbin%2Fisurf%2Dnoframes%2Fkeyword%5Fsearch%2Ecgi%3Fdb%3D%26q%3D%252Bvbthread">Image Surfer</a> | 
<a href="/go/to?tag=ex.sn.sr.6.seek&destUrl=http%3A%2F%2Fwww%2Esearch%2Ecom%2FInfoseek%2F1%2C135%2C0%2C0200%2Ehtml%3FQUERY%3D%252Bvbthread%26COLL%3DWW">Infoseek</a> | 
<a href="/go/to?tag=ex.sn.sr.6.lcos&destUrl=http%3A%2F%2Fwww%2Elycos%2Ecom%2Fcgi%2Dbin%2Fpursuit%3Fquery%3D%252Bvbthread">Lycos</a> | 
<a href="/go/to?tag=ex.sn.sr.6.yhoo&destUrl=http%3A%2F%2Fsearch%2Eyahoo%2Ecom%2Fbin%2Fsearch%3Fp%3D%252Bvbthread">Yahoo</a> |
<a href="/directory/category/0,16,home-1269,00.html?st.sn.sr.6.more">More...</a></font></td></tr>
<tr><td height=5><spacer type=block width=1 height=5></td></tr>
</table>


<!-- c|net Thu Sep 30 16:00:07 1999  $Revision: 2.29 $  -->


 
 

<table width="100%" cellspacing=0 cellpadding=0 border=0>
<tr><td bgcolor="#666666" height=2 colspan=3><spacer type=block width=1 height=1></td></tr>
<tr><td width="2%" rowspan=3><img src="/main/images/logos/logo_footer_fl.gif" width=105 height=20 border=0 alt="Snap.com"></td><td width="96%" align=center height=27><font face="MS Sans Serif, Geneva" size=-2><a href="/main/door/0,1,home-home,00.html?st.sn.ft.0.hm">Home</a>&nbsp;&nbsp;<a href="/search/directory/results/1,61,home-0,00.html?st.sn.ft.0.srch">Search</a>&nbsp;&nbsp;<a href="/main/help/item/0,11,home-8406,00.html?st.sn.ft.0.mmbr">Membership Info</a>&nbsp;&nbsp;<a href="/main/help/comment/0,98,home-0,00.html?st.sn.ft.0.fdbk">Feedback</a>&nbsp;&nbsp;<a href="/main/help/index/0,79,home-0,00.html?st.sn.ft.0.help">Help</a>&nbsp;&nbsp;<a href="/main/help/snapwire/0,445,home-0,00.html?st.sn.ft.0.wire">Free E-mail Newsletter</a></font></td><td width="2%">&nbsp;</td>
</tr><tr><td bgcolor="#CC0000" height=2 colspan=2><spacer type=block width=1 height=1></td></tr>
<tr><td align=center height=25><font face="MS Sans Serif, Geneva" size=-2><a href="/LMOID/resource/0,566,home-650,00.html?st.sn.ft.0.abt">About Snap</a>&nbsp;&nbsp;<a href="/media/index.html?st.sn.ft.o.adv">Ad Info</a>&nbsp;&nbsp;<a href="/main/help/item/0,11,home-8073,00.html?st.sn.ft.0.job">Snap Jobs</a>&nbsp;&nbsp;<a href="/main/register/problems/0,135,home-0,00.html?st.sn.ft.0.prv">Privacy Policy</a>&nbsp;&nbsp;<a href="/main/help/item/0,11,home-8450,00.html?st.sn.ft.0.surl">List Your Site</a>&nbsp;&nbsp;<a href="/main/help/item/0,11,home-8707,00.html?st.sn.ft.0.tlbx">Add Snap to Your Site</a></font></td><td><spacer type=block width=1 height=1></td></tr></table>

<p>
<!-- c|net Thu Sep 30 15:53:01 1999  $Revision: 2.29 $  -->


<font face="Arial,Helvetica" size=-2>Copyright &copy; 1997-1999 Snap and its content providers. All rights reserved. <a href="/LMOID/resource/0,566,home-764,00.html">Terms of Use and Copyright Info</a>.</font>

</body
</html>


