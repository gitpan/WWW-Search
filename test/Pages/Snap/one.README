This set of test-result pages was created on Fri Oct  8 11:06:38 1999
