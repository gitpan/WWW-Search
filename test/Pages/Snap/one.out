http://ftp.sunet.se/pub3/vendor/microsoft/Softlib/MSLFILES
http://www2.rad.kumc.edu/share
http://rs2.ch.liv.ac.uk/~dlc/Microsoft.txt
http://ap01.physik.uni-greifswald.de/~ftp/internet/cica95.idx
